website/docs/getting-started/configuration/agent.md

```diff
@@ -4,158 +4,103 @@ sidebar_position: 2
-    
-    // by default Hutch detaches from the wfexs process once it starts it,
-    // to free up threads.
-    // This forces Hutch to keep the executing thread attached to the wfexs process
-    // which means you can see stdout/stderr from wfexs in realtime,
-    // and better understand the circumstances under which wfexs exited.
-    // Intended for dev/test use while executing one job at a time.
-    "RemainAttached": false,
-  },
-
-  // Connection strings for different services
-  "ConnectionStrings": {
-    // The database tracking the jobs in the agent
-    "AgentDb": "Data Source=HutchAgent.db"
-  },
-
-  // Configurable details to add to published Results Crates.
-  "CratePublishing": {
-    // this section defaults to `null`
-    "Publisher": {
-      "Id": "" // Desired Identifier (typically URL) for the Publisher in Results Crates.
-    },
-    // this section defaults to `null`
-    "License": {
-      "Uri": "", // A URI to the license e.g. https://spdx.org/licenses/CC-BY-4.0
-      "Properties": {
-        "Identifier": null, // short-form of the license e.g. CC-BY-4.0
-        "Name": null // long-form name of the license e.g. Creative Commons Attribution 4.0 International
-      }
-    }
-  },
-
-  // This section tells Hutch where the controller API is hosted
-  "ControllerApi": {
-    // The URL of the controller API
-    "BaseUrl": ""
-  },
-
-  // Development Flags
-  // These are really intended for development or debugging use
-  // and their continued presence cannot be relied upon from one build to the next!
-  "Flags": {
-    "StandaloneMode": false, // Hutch will skip TRE Controller interactions
-    "RetainFailures": false // Hutch will not clean up working directories or database records for jobs that fail.
-  }
-}
-```
+## Production deployment
+
+Below is the service file for running Hutch as a service. The required settings are on lines that do not start with `#`. Optional settings are on lines that start with `#`. To configure a setting, add the value you wish to use immediately after the `=` without a space. For optional settings, remove the `#` from the start of the line as well.
+
+### Configuring Hutch
 
-## Guidance
-### Intermediary Store
-- Primarily for Standalone mode or as a fallback; you may configure MinIO connection details to a default store here.
+1. Set `WorkingDirectory` to the directory where HutchAgent is installed, `/home/alice/HutchAgent`.
+2. Set `User` to the username that will be running HutchAgent, e.g. `alice`.
 
-### Workflow Executor
-- The `ExecutorPath` must be the directory where WfExS is installed.
+Now you need to set the environment variables needed to run HutchAgent. These are on the lines starting with `Environment=`.
 
-- The `VirtualEnvironmentPath` must be the path to the `activate` script in the WfExS install directory, e.g. `/path/to/WfExS-backend/.pyWEenv/bin/activate`.
+1. Set `Queue__Hostname` to the URL of your queue host, e.g. `https://my-rabbitmq.example.com`.
```

Comment:
don't like this.

Environment variables are not the only way to configure it. The old section was fine for outlining the possible options and their defaults?

This is more like ansible documentation?